android/src/com/frostwire/android/gui/fragments/TransfersFragment.java

```diff
@@ -228,15 +231,26 @@ public View getHeader(Activity activity) {
         TextView text = (TextView) header.findViewById(R.id.view_transfers_header_text_title);
         text.setText(R.string.transfers);
 
-        ImageButton buttonMenu = (ImageButton) header.findViewById(R.id.view_transfers_header_button_menu);
+        buttonMenu = (ImageButton) header.findViewById(R.id.view_transfers_header_button_menu);
```

Comment:
you can use `findView(header, R.id.view_transfers_header_button_menu)` and above too (if you don't mind).